.eslintrc

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "airbnb-base",
+  "rules": {
+    "no-console" : 0,
+    "linebreak-style": 0,
+    "no-else-return":"off",
```

Comment:
Excellent work! However, I'd want you to remove the `no-else-return` rule. 

I can see that you put it there because of lines 9 and 10 of your `filter.js`. You actually don't need an `else` statement as well as the curly braces. javascript defaults it to an else if you remove the `else`.

Reply:
Also, don't disable console throughout the repository. Go to the files where you used console statements and type `/* eslint-disable no-console */` on the first line of those files. We are only disabling them in those files because we generally don't want to make it a practice to use console statements, so we will only disable in a few places.

Reply:
Okay. I have fixed them